src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier color = new ColorSupplier();
+
+    public String getRandomBall() {
+        int number = new Random().nextInt(100);
+        return new Ball(color.getRandomColor(),Integer.valueOf(number).toString()).toString();
```

Comment:
use IDEA's auto formatter (Ctrl Alt L), there should be a space after a comma
```suggestion
        return new Ball(color.getRandomColor(), number);
```